__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -11,6 +11,7 @@ const chatSchema = new ChatSchema({
   createdAt: 'createdAt',
   updatedAt: 'updatedAt',
   rawJson: {},
+  attachmentJson: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` field is initialized as an array, but in other parts of the code it seems to be treated as a string (e.g., line 250). This could lead to inconsistencies and potential errors. Please ensure that the data type is consistent across all uses.
```diff
-   attachmentJson: [],
+   attachmentJson: '',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->